packages/wdio-browserstack-service/src/service.ts

```diff
@@ -121,7 +121,7 @@ export default class BrowserstackService implements Services.ServiceInstance {
      */
     afterScenario (world: Frameworks.World) {
         const status = world.result?.status.toLowerCase()
-        if (status === 'skipped') {
+        if (status !== 'skipped') {
```

Comment:
This change seems odd to me. Someone before made this code intentionally check for `skipped`. I am afraid with this patch that we introduce a regression for something else.

Reply:
@christian-bromann as I said in the bug #7740 , the option ```preferScenarioName``` that has the purpose of display the scenario's name as session name in Browserstack is not working with the previous code, since it works just when scenario has status skipped, that no make sense. I checked in the history of this code and the change was made when you change the code to typescript, maybe you made a mistake when changed this part of the code.